test/other/embind_tsgen_memory64.d.ts

```diff
@@ -1,3 +1,7 @@
-export interface MainModule {
+interface WasmModule {
+}
+
+interface EmbindModule {
   longFn(_0: bigint): bigint;
 }
+export type MainModule = WasmModule & EmbindModule;
```

Comment:
What does `WasmModule & EmbindModule` mean here?

Reply:
It combines the interfaces of WasmModule and EmbindModule into one and exports it as MainModule. We could generate these all in one interface, but then we'd need some way of injecting output of embind into the wasm interface or vice versa. Seems cleaner to just keep them separate.